microservices/fe/thread_request.py

```diff
@@ -18,8 +18,7 @@ class Request_api (threading.Thread):
             framename(str): a timestanp used to named the frame
             thread_id(int): the id of a given thread
             t_Lock(:obj: int): the semaphore to control the number of threa opened
-            isSave_Frame(bool) : defin if the frame should be saved in disk or not.
-        
+            isSave_Frame(bool) : defin if the frame should be saved in disk or not.       
```

Comment:
As I commented elsewhere, we probably want to change this to underscores.

Reply:
done